admin-dev/themes/new-theme/js/pages/product/edit/product-form-model.js

```diff
@@ -118,15 +118,15 @@ export default class ProductFormModel {
 
     switch (event.modelKey) {
       case 'product.price.priceTaxIncluded': {
-        const priceTaxIncluded = new BigNumber(this.getProduct().price.priceTaxIncluded);
+        const priceTaxIncluded = new BigNumber(this.getProduct().price.priceTaxIncluded.replace(/,/g, '.'));
```

Comment:
This will replace commas by dots no matter the user's language right?

Reply:
yes

Reply:
It seems a bit like a quick dirty fix 😅  ?

If we agree on the method, I wish we would have a generic way to format any given price input from the DOM in a model-compliant data.

Example
```
        const priceInput = this.fetchFromDomAndProcessDataToMakeItUniform(this.getProduct().price.priceTaxIncluded);
        const priceTaxIncluded = new BigNumber(priceInput);
```